editor/modes/visual-editor/block.js

```diff
@@ -129,7 +128,8 @@ class VisualEditorBlock extends wp.element.Component {
 					}
 				},
 				...blocksWithTheSameType.slice( 1 )
-			]
+			],
+			{ offset: -1 }
```

Comment:
Hm, otherwise it would trigger an update twice, while it should be done at the same time?